app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,887 @@
+#!/usr/bin/env python
+
+'''
+Parallel execution of RTC Workflow
+'''
+
+import argparse
+from itertools import repeat
+import logging
+import multiprocessing
+import os
+import subprocess
+import time
+import yaml
+
+import isce3
+import numpy as np
+from osgeo import gdal
+
+from rtc.runconfig import RunConfig
+import rtc_s1
+
+logger = logging.getLogger('rtc_s1')
+
+def get_rtc_s1_parser():
+    '''Initialize YamlArgparse class and parse CLI arguments for OPERA RTC.
+    Modified after copied from `rtc_s1.py`
+    '''
+    parser = argparse.ArgumentParser(description='',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('run_config_path',
+                        type=str,
+                        default=None,
+                        help='Path to run config file')
+
+    # Determine the default # of concurrent workers
+    ncpu_default = os.cpu_count()
+    if os.getenv('OMP_NUM_THREADS') is not None:
+        omp_num_threads = int(os.getenv('OMP_NUM_THREADS'))
+        ncpu_default = min(ncpu_default, omp_num_threads)
```

Comment:
Are you using this value `ncpu_default`?

Reply:
get_rtc_s1_parser() in rtc_s1_parallel.py will be removed in this iteration. The default number of processes will be determined inside `run_parallel()` in `rtc_s1_parallel.py`. if the `num_process` in runconfig is `0`, then the workflow will determine the number automatically.

```python
# `run_parallel()` in `rtc_s1_parallel.py`

# determine the number of the processors here
    num_workers = cfg.groups.processing.num_process

    if num_workers == 0:
        # Decide the number of workers automatically
        ncpu_system = os.cpu_count()
        omp_num_threads = os.getenv('OMP_NUM_THREADS')
        if omp_num_threads:
            num_workers = min(ncpu_system,
                              omp_num_threads,
                              len(list_burst_runconfig))
```